src/test/scripts/product_versions.py

```diff
@@ -45,6 +53,13 @@ def get_lts_version(argv):
 	return lts_version
 
 
+def loadJSON(fdata):
+	result = re.search("\[.*\]", fdata.decode("utf-8"))
+	if result is None:
+		return ""
```

Comment:
Since this is returning a list of values, shouldn't it return `[]` in the error case?

Reply:
I tested with invalid data and returns None

Reply:
`result` is `None` but you specifically return `""` if `result` is `None`. In the code above, you have the line

```
feeds += loadJSON(current_feeds)
```

If `loadJSON` returns `""`, it'll append an empty string to a list, and I think python will coerce the value into a string, which is probably not what you want.

```suggestion
		return []
```